asapdiscovery-docking/asapdiscovery/docking/scripts/dock_small_scale_end_to_end.py

```diff
@@ -259,10 +260,11 @@
 )
 
 parser.add_argument(
-    "--target",
+    "--viz-target",
```

Comment:
I have changed the name of this argument to `--viz-target` so its meaning is clearer.